rubocop-rspec.gemspec

```diff
@@ -36,8 +36,7 @@ Gem::Specification.new do |spec|
     'documentation_uri' => 'https://docs.rubocop.org/rubocop-rspec/'
   }
 
-  spec.add_runtime_dependency 'rubocop', '~> 0.87'
-  spec.add_runtime_dependency 'rubocop-ast', '>= 0.7.1'
+  spec.add_runtime_dependency 'rubocop', '~> 1.0'
```

Comment:
I believe we also need `rubocop-ast` 1.1.0 for DSL aliases's `Module.method` syntax. @bquorning

Reply:
I'm sorry I've overlooked this in #956

Reply:
Hmm, which of these, then?

1. `'>= 1.1.0'`
2. `'~> 1.1.0'`
3. `'~> 1.1'`

I think 3 should be fine, right?

Reply:
1 is the best.